src/main/java/newbank/server/NewBank.java

```diff
@@ -163,4 +156,29 @@ private String newAccount(final Customer customer, final String accountName) {
         .findFirst()
         .map(e -> e.getValue());
   }
+
+  private Optional<Account> getAccount(final Customer customer, final String accountName) {
+    return customer.getAccount(accountName);
+  }
+
+  /**
+   * Deposit some money into a specified customer's account
+   *
+   * @param customerID The customer identifier
+   * @param account The account name
+   * @param money The amount to deposit
+   */
+  public synchronized void depositMoney(
+      final CustomerID customerID, final String accountName, final Money money)
+      throws AccountInvalidException {
+
+    Customer customer = customers.get(customerID.getKey());
+    Optional<Account> account = getAccount(customer, accountName);
+
+    if (account.isEmpty()) {
+      throw new AccountInvalidException();
+    }
+
+    account.get().addMoney(money);
```

Comment:
`account.isEmpty()` is causing compiler errors for me. I'm a bit confused as to how it's working for everyone else!
`Optional#isEmpty` was introduced in Java 11, but the project uses Java 8, so it *should* cause an error.
These lines can be replaced with the Java-8-compatible (and idiomatic):
```java
account.orElseThrow(AccountInvalidException::new).addMoney(money);
```
I'll add a PR, but someone tell me if I'm being crazy!

Reply:
Hi Zac,
Yes that looks good.